### DISCUSSION TITLE
Feature: social sharing button on single post pages

### DISCUSSION THREAD

**Discussion by lirantal** [arthelokyo/astrowind#48] [Ideas]
It's common for blogs to feature social sharing buttons for some of the popular mediums and show them on each blog post (at the top, bottom, or other layout options).

**Comment by prototypa** [comment]
Hi @lirantal 

In this https://github.com/onwidget/astrowind/commit/000a99a3fcbe313bf3a438786ea20357732e0960 we have added **Social Share Buttons.** 

We preferred in this case not to use your PR because it used links in <a> and maybe this could create some SEO problems.

For now the buttons are only at the bottom of the post. We have been thinking about doing something very similar to Medium, but for this we would have to restructure the blog design a bit.

There is also an idea that @yuxxeun brought to the table that sounds interesting: https://github.com/onwidget/astrowind/discussions/47

**Reply by lirantal** [reply]
Ok
